ace-internal.d.ts

```diff
@@ -1554,7 +1564,3 @@ declare module "./src/mouse/default_gutter_handler" {
     }
 }
 
-declare module "./src/lib/keys" {
```

Comment:
Missed this during the initial review, what was the reason for removing the type declaration of `lib/keys`?

Reply:
Actually that was wrong, It exported only one function without repeating structure for this module, which is wild mix of 
```javascript
oop.mixin(exports, Keys);
exports.default = exports;
exports.keyCodeToString
```